packages/app-utils/src/storage/LocalWorkspaceStorage.ts

```diff
@@ -57,13 +57,18 @@ export class LocalWorkspaceStorage implements WorkspaceStorage {
       showEmptyStrings: true,
       showNullStrings: true,
       showExtraGroupColumn: true,
-      defaultNotebookSettings: {
+      notebookSettings: {
```

Comment:
I changed it because nothing about this is "default" IMO. It's the user configuration. We define the defaults in `LocalWorkspaceStorage`. So to me the name didn't imply what this actually did and was confusing what the actual default value was.

It is technically breaking, but minimap isn't available in DHE. It's an easy enough toggle that I think it is worth it for clearer naming